migrations/versions/ed7bf6adbd4d_add_alert_template.py

```diff
@@ -0,0 +1,28 @@
+"""add_alert_template_column
+
+Revision ID: ed7bf6adbd4d
+Revises: 71477dadd6ef
+Create Date: 2018-11-28 22:56:40.494028
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'ed7bf6adbd4d'
+down_revision = '73beceabb948'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('alerts', sa.Column('template', sa.Text(), nullable=True))
```

Comment:
Let's put `template` in the `options` JSON object we already have in `alerts`.

Reply:
okay, fix logic and remove migration file.